pkg/actors/actors.go

```diff
@@ -81,6 +82,7 @@ type Actors interface {
 	Call(ctx context.Context, req *invokev1.InvokeMethodRequest) (*invokev1.InvokeMethodResponse, error)
 	Init(context.Context) error
 	GetState(ctx context.Context, req *GetStateRequest) (*StateResponse, error)
+	GetBulkState(ctx context.Context, req *GetBulkStateRequest) (BulkStateResponse, error)
```

Comment:
This is exactly the type of API I was hoping we could add to improve efficiency. Glad to see it here!